### PR TITLE
Create 2020-06-17-developer-workshop-bitcoin-lightning.md

### PR DESCRIPTION
**Pull request by basilesportif**
@mikeosborne check this

### CODE REVIEW

**Comment by mikeosborne** [comment]
We need the youtube link for the event.

File: content/events/2021-06-17-developer-workshop-bitcoin-lightning.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
++++
+
```
Comment (jalehman) [inline content/events/2021-06-17-developer-workshop-bitcoin-lightning.md:2]: Remove newline

File: content/events/2021-06-17-developer-workshop-bitcoin-lightning.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+date = 2021-06-17 
+[extra] 
+pinned = false 
+host = "Tim (~timluc-miptev)" 
+time = "10:00 PST" 
+location = "Online Event" 
+image = "" 
+registration_url = "" 
+description = "Programming Bitcoin and Lightning on Urbit" 
+
```
Comment (jalehman) [inline content/events/2021-06-17-developer-workshop-bitcoin-lightning.md:13]: Remove newline

File: content/events/2021-06-17-developer-workshop-bitcoin-lightning.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
++++
+
+title = "Developer Workshop: Programming Bitcoin on Urbit"
+date = 2021-06-17 
+[extra] 
+pinned = false 
+host = "Tim (~timluc-miptev)" 
+time = "10:00 PST"
```
Comment (jalehman) [inline content/events/2021-06-17-developer-workshop-bitcoin-lightning.md]:
```suggestion
time = "09:00 PDT" 
```

File: content/events/2021-06-17-developer-workshop-bitcoin-lightning.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+location = "Online Event" 
+image = "" 
+registration_url = "" 
+description = "Programming Bitcoin and Lightning on Urbit" 
+
++++
+
+Programming Bitcoin is fun, particularly on Urbit, but the devil is in the details. Tim (~timluc-miptev) will impart the implicit knowledge he gained working with Bitcoin in the course of his projects, and give an open workshop on what you need to know, either to satisfy your curiosity or to jump in and help. Tim will give a structured presentation on low-level Bitcoin fundamentals as well as Lightning and Taproot, intended to make it easy to use Bitcoin resources for further study.
+
+Tim is the technical director at the Urbit.org Foundation, and wrote the backend for the Urbit Bitcoin wallet. He is currently managing `volt`, a project to integrate a Lightning client into Urbit.  This workshop will be interactive--Tim will take questions throughout, and will conduct another session later if material remains.
```
Comment (jalehman) [inline content/events/2021-06-17-developer-workshop-bitcoin-lightning.md]:
```suggestion
Tim is the technical director at the Urbit.org Foundation, and wrote the backend for the Urbit Bitcoin wallet. He is currently managing `volt`, a project to integrate a Lightning client into Urbit.  This workshop will be interactive&mdash;Tim will take questions throughout, and will conduct another session later if material remains.
```

**Review by jalehman — APPROVED** [review]
Some minor changes.

**Comment by jalehman** [comment]
@mikeosborne this has no video link—can you add that please?

**Pull request merged**